deeppavlov/models/morpho_tagger/network.py

```diff
@@ -63,11 +87,13 @@ def __init__(self, symbols: DefaultVocabulary, tags: DefaultVocabulary,
         self.word_lstm_units = word_lstm_units
         self.regularizer = regularizer
         self.verbose = verbose
-        self.initialize()
-        log.info("{} symbols, {} tags in CharacterTagger".format(self.symbols_number_, self.tags_number_))
+        self._initialize()
         self.build()
 
-    def initialize(self):
+    def _initialize(self):
+        """
```

Comment:
It is not mandatory to make docstrings for private methods. Moreover, obvious docstrings are useless.
One-line docstring looks better when quotes are also in this line.